tests/workflows/unit_tests/core_steps/transformations/test_crop.py

```diff
@@ -48,17 +52,20 @@ def test_crop_image() -> None:
     # given
     image = np.zeros((1000, 1000, 3), dtype=np.uint8)
     origin_size = {"height": 1000, "width": 1000}
-    predictions = [
-        {"x": 10, "y": 10, "width": 20, "height": 20, "detection_id": "one"},
-        {"x": 100, "y": 100, "width": 40, "height": 40, "detection_id": "two"},
-        {"x": 500, "y": 500, "width": 100, "height": 100, "detection_id": "three"},
-    ]
+    predictions = convert_to_sv_detections(predictions=[{
```

Comment:
please create sv.Detections directly, as we do not want faulty `convert_to_sv_detections` impacting other unit tests

Reply:
the same applies for all `convert_to_sv_detections(...)` usage in tests apart from tests of `convert_to_sv_detections` itself

Reply:
Done